src/index.html

```diff
@@ -38,30 +38,36 @@
     <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
     <script type="text/javascript">
         /* Specify the URL of the source that the home application loads. */
-        var appUnitFQDN = "demo.personium.io";
-        var appMarketListEndpoint = "https://market.demo.personium.io/OData/applist/Apps";
-        var appDirectoryEndpoint = "https://directory.demo.personium.io/app-uc-directory/OData/directory";
-        var homeAppUrl;
+        // var appCellName = "${CELL_NAME}";
+        // var appUnitFQDN = "${UNIT_FQDN}";
+        var appMarketListEndpoint = "${MARKETLIST_ENDPOINT}";
+        var appDirectoryEndpoint = "${APPDIRECTORY_ENDPOINT}";
         var appUseBox = "__";
-        $.ajax({
-            type: "GET",
-            url: "https://" + appUnitFQDN + "/",
-            headers: {
-                'Accept': 'application/json'
-            }
-        }).done(function (unitObj, status, xhr) {
-            let ver = xhr.getResponseHeader("x-personium-version");
-            if (ver < "1.7.1" || unitObj.unit.path_based_cellurl_enabled) {
-                homeAppUrl = "https://" + appUnitFQDN + "/app-cc-home/";
-            } else {
-                homeAppUrl = "https://app-cc-home." + appUnitFQDN + "/";
-            }
-        }).fail(function (res) {
-            homeAppUrl = "https://" + appUnitFQDN + "/app-cc-home/";
-        }).always(function () {
-            loadStyleSheet();
-            loadScript();
-        });
+        // $.ajax({

```

Comment:
If you remove this part, will HomeApp still being able to differentiate path based or not?

Reply:
If you deploy HomeApp to path-based Unit, you can set path-base URL on `HOME_APP_CELL_URL`.

I don't think there is no reason to constrain to deploy HomeApp as Personium App. So, I think user should be able to set url which is not specified by Personium Cell. Let's discuss about that.

Reply:
I make some code changed to enable you to choose "specifying `CELL_NAME` and `UNIT_FQDN`" or "specifying `HOME_APP_CELL_URL`". If you don't specify `HOME_APP_CELL_URL`, HomeApp uses `CELL_NAME` and `UNIT_FQDN` to compose CellUrl of HomeApp.

Reply:
Thank you! It is great to have dynamic setup.